packages/frontend/src/components/NavBar/ExploreMenu/index.tsx

```diff
@@ -1,64 +1,170 @@
 import {
     Button,
-    Menu,
+    Icon,
+    IconName,
     PopoverInteractionKind,
     Position,
-    Spinner,
 } from '@blueprintjs/core';
 import { Popover2 } from '@blueprintjs/popover2';
 import { subject } from '@casl/ability';
-import { FC, memo } from 'react';
+import { FC, memo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useCreateMutation } from '../../../hooks/dashboard/useDashboard';
+import { DEFAULT_DASHBOARD_NAME } from '../../../pages/SavedDashboards';
 import { useApp } from '../../../providers/AppProvider';
 import { Can } from '../../common/Authorization';
-import NavLink from '../../NavLink';
-import { FirstItem, NavbarMenuItem, SpinnerWrapper } from '../NavBar.styles';
+import SpaceActionModal, { ActionType } from '../../common/SpaceActionModal';
+import {
+    ButtonWrapper,
+    HelpItem,
+    IconContainer,
+    ItemCTA,
+    ItemDescription,
+    MenuWrapper,
+} from './ExploreMenu.styles';
 
 interface Props {
     projectUuid: string;
 }
 
+interface ExploreItemProps {
+    icon: IconName;
+    title: string;
+    description: string;
+}
+const ExploreItem: FC<ExploreItemProps> = ({ icon, title, description }) => {
+    return (
+        <HelpItem>
+            <IconContainer>
+                <Icon icon={icon} />
+            </IconContainer>
+            <div>
+                <ItemCTA>{title}</ItemCTA>
+                <ItemDescription>{description}</ItemDescription>
+            </div>
+        </HelpItem>
+    );
+};
 const ExploreMenu: FC<Props> = memo(({ projectUuid }) => {
     const { user } = useApp();
+    const history = useHistory();
+    const [isOpen, setIsOpen] = useState<boolean>(false);
+    const {
+        isLoading: isCreatingDashboard,
+        isSuccess: hasCreatedDashboard,
+        mutate: createDashboard,
+        data: newDashboard,
+        reset,
+    } = useCreateMutation(projectUuid);
+    const [isCreateSpaceOpen, setIsCreateSpaceOpen] = useState<boolean>(false);
 
+    if (!isCreatingDashboard && hasCreatedDashboard && newDashboard) {
+        history.push(
+            `/projects/${projectUuid}/dashboards/${newDashboard.uuid}`,
+        );
+        reset();
+    }
     return (
-        <Popover2
-            interactionKind={PopoverInteractionKind.CLICK}
-            content={
-                !projectUuid ? (
-                    <SpinnerWrapper>
-                        <Spinner size={20} />
-                    </SpinnerWrapper>
-                ) : (
-                    <Menu>
-                        <NavLink to={`/projects/${projectUuid}/tables`}>
-                            <FirstItem
-                                roleStructure="menuitem"
+        <>
+            <Popover2
+                isOpen={isOpen}
```

Comment:
At the moment the popup seems to be persistent. 


![persistent popup](https://user-images.githubusercontent.com/9117144/206474638-11535e12-abf0-4284-8971-aa814f43de57.gif)